docs/docs/dev/explore/dpp-on-kubernetes.md

```diff
@@ -116,3 +116,147 @@ On Kubernetes, `Dataplane` resource is automatically created by kuma-cp. For eac
 
 When a Pod is deleted its matching `Dataplane` resource is deleted as well. This is possible thanks to the
 [owner reference](https://kubernetes.io/docs/concepts/overview/working-with-objects/owners-dependents/) set on the `Dataplane` resource.
+
+## Custom Container Configuration
+
+For the vast majority of situations, using the default configuration of injected
+by Kuma containers (`kuma-sidecar` and `kuma-init`) is suggested. If you want to
```

Comment:
Maybe move this in a tip.

Something like:

::: warning
In the vast majority of cases you shouldn't need to override the sidecar and init-container configuration. container-patches is a feature which requires good understanding of both Kuma and Kubernetes.
:::

Reply:
addressed